internal/pkg/plugin/trellogithub/github.go

```diff
@@ -183,3 +183,28 @@ func (gi *TrelloGithub) AddTrelloIdSecret(trelloId *TrelloItemId) error {
 
 	return nil
 }
+
+func (gi *TrelloGithub) GetWorkflowState() (map[string]interface{}, error) {
```

Comment:
Same as the other one, maybe a better naming could improve the readability of the code.